readme.md

```diff
@@ -39,6 +39,14 @@ details.branchName // is null if the repository in detached HEAD mode
 details.isCleanTag
 ```
 
+You can optionally search a subset of tags with `prefix`. Inclusion of the `prefix` in the returned version is toggled
+with `includePrefix` (default: `false`). Example when the tag is my-product2.15.0:
+
+```groovy
+gitVersion(prefix:'my-product') // -> 2.15.0
+gitVersion(prefix:'my-product', includePrefix:true) // -> my-product2.15.0
```

Comment:
Rationale: the tooling should guide users towards standard best-practices. For us, they are to use X.Y.Z, not vX.Y.Z.

Let's also constrain the prefix a bit more, how about this: [a-z]+([a-z]+[-/])*@
Good:
```
abc@
abc@
foo-bar@
foo/bar@
```

Bad
```
foo-
foo@bar@
```